util/test/checkChplInstall

```diff
@@ -96,6 +96,26 @@ else
     CHPL_DIR_REMOVE=0
 fi
 
+# Tmp directory for compile output
+TMP_TEST_DIR="$(mktemp -d ${CHPL_DIR}/chapel-test-XXXXX)"
+log_info "Temporary test job directory: ${TMP_TEST_DIR}"
+
+# Cleanup the tmp directory whenever exit is invoked
+function cleanup_tmp_dir()
+{
+    log_debug "Removing ${TMP_TEST_DIR}"
+    rm -rf ${TMP_TEST_DIR} || (
+        log_warning "failed Removing ${TMP_TEST_DIR}, ignore the error"
+        set -x; ls -la ${TMP_TEST_DIR}
+    ) || : ok
```

Comment:
Right - I understand about || (although your explanation is helpful) - the part I'm really unclear on is : ok. I've never seen : before in a shell script. Does it introduce a comment? Is ok a command?